Kaos-Frontend-Coding-Standards.md

```diff
@@ -74,7 +74,7 @@ At Sonatype we value the stability and maintainability of the code base while st
   * baz.view.html <-> baz.controller.js
 
 ## AngularJS Development
-* Each Angular Component should exist in it's own file
+* Each Angular Component should exist in it's own file, if it is independent
```

Comment:
I think the directive example below helps clarify the meaning of 'dependent', which should also clarify 'independent'.